universal-application-tool-0.0.1/app/views/applicant/ProgramIndexView.java

```diff
@@ -90,17 +90,22 @@ private ContainerTag topContent(String titleText, String infoTextLine1, String i
                 StyleUtils.responsiveSmall(Styles.TEXT_5XL),
                 Styles.FONT_SEMIBOLD,
                 Styles.MB_2,
```

Comment:
General comment, want to confirm whether there are any unit tests for coverage here

Reply:
Unfortunately it doesn't seem there are. We were saying it would be awesome if there were screenshot tests or something, but that feels like a very big change to add. Could be a good suggestion for the full time team. But let me know if you have any suggestions for how we might test this in the shorter term.